hawkular-alerts-engine/src/main/resources/org/hawkular/alerts/engine/rules/ConditionMatch.drl

```diff
@@ -293,16 +296,31 @@ rule AlertOnSatisfiedDampening
         $t  : Trigger( mode == Mode.FIRING, $tid : id )
         $d  : Dampening( triggerMode == Mode.FIRING, triggerId == $tid, satisfied == true )
     then
-        if (log != null && log.isDebugEnabled()) {
-            log.debug("AlertOnSatisfiedDampening! " + $d.log());
+        Event newEvent;

```

Comment:
It helps me to think about the issues to follow the impl impact to some degree.  I think it is just style of work.